web-client/src/styles/forms.scss

```diff
@@ -134,6 +132,14 @@ legend {
   }
 }
 
+textarea {
+  min-height: calc(
+    1em + 24px
+  ); // Don't allow resizing smaller than the line of text
```

Comment:
There is nothing sacred about this. It just seemed to look good with a variety of font sizes, and most importantly the font size we use by default.